docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -0,0 +1,189 @@
+# Get invited and onboard to TechPass
+
+This article guides you how to request for a TechPass account and onboard to it. Refer to the flow chart on [Onboard to TechPass](onboard-to-techpass) page to know if you can sign up for TechPass account via TechPass portal or not. If you can't sign up for TechPass account via TechPass portal, you need to request to get invited.  
+
+?> **Note**<br>- Alternatively, public officers with a non-SE GSIB device can access [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg) to request for TechPass and SEED(optional) provisioning. <br>- For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+
+
+## Audience
+
+Users who need a TechPass account but do not have a WOG account or a non-SE GSIB device.
+
+## Prerequisites
+
+You need the following to get invited to TechPass and complete the onboarding:
+
+- Your organisational email address provided by your organisation.
+- A standard mailbox for your organisational email address. TechPass does not support email accounts which do not have an inbox, such as LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before requesting for TechPass.
+- Before you onboard, ensure you have received the TechPass onboarding email and is still valid.
+
+?> TechPass onboarding email is valid for 30 days. If you do not onboard to TechPass within this 30 days, we will terminate your TechPass account, and you need to sign up again.
+
+
+> **Tip**: Click the triangle to view the instructions to complete each step.
+
+## Step 1. Request for TechPass account
+
+<details><summary style="font-size:20px;font-weight:bold">Get invited to TechPass account and request for SEED (optional)</summary>
+
+1. Contact your project manager or the reporting officer to request for TechPass account and SEED provisioning(optional).
+
+!> You need SEED provisioning to access SGTS and GCC 2.0 resources via an Internet Device.
+
+2. Provide the required details in this request such as your organisational email address, mobile phone number and project name.
+
+   Project manager or the reporting officer contacts the sponsoring agency or the tenant admin to invite you to TechPass.
+
+<!--3. If you are requesting for SEED provisioning, provide the details of your Internet Device.
+
+Your project manager or reporting officer will work with the sponsoring government agency to provide the requested services.
+-->
+
+> **Additional information**:
+  >
+  > **When you are invited to TechPass**
+  >- A TechPass account is provisioned for you and is in pending state.
+  >- We'll send the TechPass onboarding email to activate the account. This email is valid only for 30 days and will have your TechPass account or log in ID. 
+  >- Your TechPass log in ID's domain will be ``techpass.gov.sg```.
+  
+  >
+  > **If SEED provisioning is approved**:
```

Comment:
I'm not sure if we should use **approved** because technically the license is given without anybody approving it.

Reply:
noted, will change it to "provisioned"